test/Serialization/differentiable_attr.swift

```diff
@@ -2,8 +2,11 @@
 
 // RUN: %empty-directory(%t)
 // RUN: %target-swift-frontend %s -emit-module -parse-as-library -o %t
+// RUN: llvm-bcanalyzer %t/differentiable_attr.swiftmodule | %FileCheck %s -check-prefix=BCANALYZER
 // RUN: %target-sil-opt -disable-sil-linking -enable-sil-verify-all %t/differentiable_attr.swiftmodule -o - | %FileCheck %s
 
+// BCANALYZER-NOT: UnknownCode
```

Comment:
I'm curious about the meaning of this check?

Reply:
To make sure the code block is registered. Other attribute serialization tests have this check.